### DISCUSSION TITLE
how to buid previous version detectron2=0.3 with Mac OS.

### DISCUSSION THREAD

**Discussion by Zzrealjim** [facebookresearch/detectron2#3637] [Q&A]
Hi,
I try to run LayoutLM which requires pytorch= 1.7. So I need to install detectron2=0.3. All the wheels for the previous version are used for Linux. 

> eric@Yangs-MacBook-Pro layoutlmft % pip install layoutlmft/detectron2-0.3+cpu-cp38-cp38-linux_x86_64.whl
> ERROR: detectron2-0.3+cpu-cp38-cp38-linux_x86_64.whl is not a supported wheel on this platform.

The Mac installation can only install the newest version which needs pytorch>1.8. Anyone can help me with it?
Cheers,
Eric

**Comment by MiroPsota** [comment]
You can try my repository for building packages and PyPI simple index and see if it works for you:
https://github.com/facebookresearch/detectron2/discussions/5200
